webpack.config.js

```diff
@@ -67,5 +60,6 @@ module.exports = (env) => {
       hot: true,
       open: true,
     },
+    devtool: "source-map",
```

Comment:
`source-map` is the slowest style of source-maps. It is good only for production environment. Please, specify some faster but still good enough alternative for development.